doc/supportedPlatformVersions.md

```diff
@@ -623,6 +626,7 @@
 - 16.19.0
 - 16.20.0
 - 16.20.1
+- 16.20.2
```

Comment:
Do we need to update minor versions for node runtimes [here](https://github.com/microsoft/Oryx/blob/main/doc/supportedRuntimeVersions.md#node)?

Reply:
We do not because our Node runtime images are published with _just_ the major version, but if we ever wanted to be more granular with the runtime image tags (specifically for the buildpack case where we'd want to do runtime swapping), then we'll want to add a tag that includes the minor version, and maybe patch as well.

Reply:
Thanks for clarifying @cormacpayne :)